be/src/runtime/local_tablets_channel.cpp

```diff
@@ -445,6 +445,9 @@ void LocalTabletsChannel::add_chunk(Chunk* chunk, const PTabletWriterAddChunkReq
 }
 
 void LocalTabletsChannel::_flush_stale_memtables() {
+    if (_immutable_partition_ids.empty() && config::stale_memtable_flush_time_sec <= 0) {
```

Comment:
stale_memtable_flush_time_sec is mutable, so here create one local variable is better.

auto flush_time=config::stale_memtable_flush_time_sec